ci/scenarios/backend/be_0002_add_update_remove_preset_test/main.py

```diff
@@ -141,35 +132,35 @@ def validate_add_update_remove_preset(preset_name='integrationtests'):
         # add and remove different presets
         AddUpdateRemovePreset.LOGGER.info("Started adding and removing different presets")
         for preset_def, preset_details in presets.iteritems():
-            AddUpdateRemovePreset._add_remove_preset(alba_backend.name, preset_details, preset_def, api)
```

Comment:
AddUpdateRemovePreset can be replaced by cls
Also no need to pass api here as it is the same class which can just call cls.api in the other method